### DISCUSSION TITLE
Alpha 2 🎉

### DISCUSSION THREAD

**Discussion by prayag17** [prayag17/Blink#180] [Announcements]
## What's Changed
- Added Music Player #173
- Improved Library Page #176
- Improved Title Page #160
- Remade Card Component #153
- Improve Home Page #136
- Added Search Page #34
- Improved Sidemenu #179
- Display BoxSet #125
- Fixed tons of bugs and improved performance...

**Full Changelog**: https://github.com/prayag17/JellyPlayer/compare/v1.0.0-alpha01...v1.0.0-alpha02

<hr /><em>This discussion was created from the release <a href='https://github.com/prayag17/JellyPlayer/releases/tag/v1.0.0-alpha02'>Alpha 2 🎉</a>.</em>

**Comment by qvattr0** [comment]
I've just downloaded the Alpha 2 release, used it for a couple of minutes, and would like to express my appreciation for this piece of software. I bet that as this player is further refined, more people will find to appreciate it more than the clunky Jellyfin Media Player. Keep up the phenomenal work!
